<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="upload_file_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="real_goose_pub_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Real Goose Pub</string>
    </property>
   </widget>
   <widget class="QPushButton" name="real_goose_sub_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Real Goose Sub</string>
    </property>
   </widget>
   <widget class="QPushButton" name="easyeditor_button">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>240</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>SmartScreen</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <addaction name="menuStart"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
